Tutorials/python/samtools_count_para_reg_multiprocess_py/Readme.md

```diff
@@ -9,7 +9,7 @@ In order to take full advantage of the scalability that cloud computing offers,
 This applet tutorial code is extremely similar to the [_Parallel Threads SAMtools count tutorial_](/python_parallel_tutorial.html#samtools_count_para_chr_subprocess_py), except `multiprocessing` is used instead of `multiprocessing.dummy`.
 
 ## How is the SAMtools dependency provided?
-The SAMtools dependency is resolved by declaring an [Apt-Get](https://help.ubuntu.com/14.04/serverguide/apt-get.html) package in the `dxapp.json` `runSpec.execDepends` field.
+The SAMtools dependency is resolved by declaring an [Apt-Get](http://manpages.ubuntu.com/manpages/xenial/man8/apt-get.8.html) package in the `dxapp.json` `runSpec.execDepends` field.
 ```json
 {
   "runSpec": {
```

Comment:
In L22:  "execDepends wiki page" --> "execDepends documentation"

Reply:
done